bank-sdk/example-app/src/main/java/net/gini/android/bank/sdk/exampleapp/ui/CaptureFlowHostActivity.kt

```diff
@@ -21,7 +23,13 @@ class CaptureFlowHostActivity : AppCompatActivity() {
         if (savedInstanceState == null) {
             val openWithIntent = IntentCompat.getParcelableExtra(intent, EXTRA_IN_OPEN_WITH_INTENT, Intent::class.java)
             if (openWithIntent != null) {
-                startBankSDKForOpenWith(openWithIntent)
+                if (!intent.hasExtra(ExampleUtil.DOCUMENT)) {
+                    startBankSDKForOpenWith(openWithIntent)
+                } else {
+                    intent.getParcelableExtra(ExampleUtil.DOCUMENT, Document::class.java)?.let {
+                        startBankSDKForDocument(it)
+                    }
+                }
```

Comment:
[Suggestion] Can we separate the "open with" intent case from the "open with" document one?

I would suggest to move the "has document extra" check out of the "has open with intent" check. Also to use `IntentCompat.getParcelableExtra()` because it's backwards compatible:
```
if (openWithIntent != null) {
    startBankSDKForOpenWith(openWithIntent)
} else if (intent.hasExtra(ExampleUtil.DOCUMENT) {
    IntentCompat.getParcelableExtra(intent, ExampleUtil.DOCUMENT, Document::class.java)?.let {
        startBankSDKForDocument(it)
    }
}